.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,64 @@
+name: SonarCloud
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  info:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Dump GitHub context
+      env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: echo "$GITHUB_CONTEXT"
```

Comment:
Why dump github context ?

Reply:
You're right, it's useless here, it was juste a first debug step before running build + sonar, 
I'll commit the removal of this job